sbws/config.default.ini

```diff
@@ -23,6 +23,8 @@ http_timeout = 10
 # CircuitBuildTimeout doesn't handle the case of a TLS connection to a relay
 # taking forever, and probably other not-yet-discovered cases.
 circuit_timeout = 10
+reset_bw_ipv4_changes = on
+reset_bw_ipv6_changes = off
```

Comment:
Please leave a comment here documenting the purpose of these two options

Reply:
Please add these options to `_validate_general` in `sbws/util/config.py`. Otherwise, you get this when running sbws.

```
$ sbws generate
general/reset_bw_ipv4_changes (on): Unknown key
general/reset_bw_ipv6_changes (off): Unknown key

Reply:
Please add a comment that IPv6 trimming isn't implemented yet.
Otherwise, operators will set it to on, and be surprised when they get runtime errors.